nuxeo-tools-hooks/nxtools/hooks/services/github_service.py

```diff
@@ -337,6 +337,10 @@ def github_notify(self, pull_request):
         jira = services.get(JiraService)
         keys = jira.get_issue_ids_from_pullrequest(pull_request)
 
+        if len(keys) > 0:
```

Comment:
```suggestion
        if keys:
```
Minor improvement.

Reply:
I actually changed it :) will change it back